app/controllers/admin/tag_rules_controller.rb

```diff
@@ -9,8 +9,8 @@ class TagRulesController < ResourceController
     def map_by_tag
       respond_to do |format|
         format.json do
-          serialiser = ActiveModel::ArraySerializer.new(collection)
-          render json: serialiser.to_json
+          serializer = ActiveModel::ArraySerializer.new(collection)
```

Comment:
Oh no, I like the UK spelling! :stuck_out_tongue: I wish we could convince the Rails team to switch to Serialisers...

Oh well, this is probably a good move then. Does that mean we will switch to American English in the whole code base for consistency? And documentation? No! :fearful:

Reply:
tbh all I care about is consistency, I think we must be consistent in each word. We must not have a word spelled in two different ways in our codebase.

From my time coding in London with English and Australians devs I remember well that Australians were a lot more pro British spelling than the English themselves :-)

Reply:
Yeah, and Germans living in Australia are even worse! :wink: